spec/services/employees/member_creator_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Employees::MemberCreator, type: :controller do
+  let(:params) { user_params }
```

Comment:
what is need to use let here? we can simply use user_params (but it's not obvious for tests to move it out to separate module) I'd leave it in the spec file.